src/app/presentation/components/home-components/about/about.component.html

```diff
@@ -31,9 +31,9 @@ <h1 class="registration_title">About Us</h1>
               <mat-grid-tile>
                   <div class="about-text text-left justify-start">
                       <h3 class="feature-title">About team</h3>
-                      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
-                      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore.</p>
-                      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                      <p>We are the young team, we are from different cities, we have different education, but we are united by one goal – to become the great developers.</p>
+                      <p>We have known each other in all for three month, but despite this we are choosing, planning, and starting the development of our project. Many of using technologies and frameworks was new for us, and we spend lot of time on the beginning to investigate it. And additionally nobody worked with SCRUM before. But we have done a great job, and you will see that from first minutes in our application</p>
+                      <p>We using many technologies, such as: Angular, C#, Entity Framework, SQL, ASP.NET, GIT, Azure. Most of them was new for us, but we done it wery well. </p>
```

Comment:
Many technologies were used, such as: Angular, C#, Entity Framework, SQL, ASP.NET, GIT, Azure. We have done this project, despite the fact that it was new to us.